x-pack/plugins/apm/common/anomaly_detection/get_anomaly_detection_setup_state.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { FETCH_STATUS } from '../../public/hooks/use_fetcher';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import type { APIReturnType } from '../../public/services/rest/createCallApmApi';
+import { ENVIRONMENT_ALL } from '../environment_filter_values';
```

Comment:
Why is this file in `/common` if it's only used by on the client?  Looks like those eslint warnings would go away if this was moved to `/public`